src/CommonAssemblyInfo.cs

```diff
@@ -32,13 +32,13 @@
 [assembly: AssemblyTrademark("NUnit is a trademark of NUnit Software")]
 
 #if DEBUG
-#if NET_4_5
+#if NET45
```

Comment:
These defines are automatically included now and this is the default format, so I switched all code to use the defaults.